From 299fc807efc26c5c103220e2c6bf80b851917fe2 Mon Sep 17 00:00:00 2001
From: Sergey Bolshakov <sbolshakov@altlinux.org>
Date: Tue, 20 Nov 2007 14:57:52 +0300
Subject: [PATCH 06/17] Revert "	* generic/tclObj.c: remove superfluous
 #include of tclCompile.h"

This reverts commit 267adc009b1ac3d1c5bdf495eb434b0503d744f8.
---
 tcl/generic/tclObj.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tcl/generic/tclObj.c b/tcl/generic/tclObj.c
index 255614a60e..bead2b64aa 100644
--- a/tcl/generic/tclObj.c
+++ b/tcl/generic/tclObj.c
@@ -15,6 +15,7 @@
  */
 
 #include "tclInt.h"
+#include "tclCompile.h"
 #include "tommath.h"
 #include <math.h>
 
-- 
2.21.0

